test/e2e/app/app_test.go

```diff
@@ -113,6 +113,16 @@ var _ = ginkgo.Describe("App", func() {
 			gomega.Ω(appCRD.Spec.TaskGroups[0].Tolerations[0].Effect).To(gomega.Equal(v1.TaintEffectNoSchedule))
 		})
 
+		ginkgo.It("Verify the Yunikorn Scheduler healthy", func() {
+			ginkgo.By("Call the HealthCheck API")
+			healthCheck, err := yunikorn.GetHealthCheck()
+			gomega.Ω(err).NotTo(gomega.HaveOccurred())
+			gomega.Ω(healthCheck.Healthy).To(gomega.BeTrue())
+			for _, check := range healthCheck.HealthChecks {
+				gomega.Ω(check.Succeeded).To(gomega.BeTrue())
+			}
+		})
```

Comment:
I am not very familiar with ginkgo framework, but since this is the same function call for every test, can we move this to a utility function and reuse the code everywhere? If that is possible.

Reply:
Instead of making this a standalone test for each suite, include the health check as part of the AfterEach(). Should not be much overhead for just a rest call, and this will make it easier to pinpoint after which test Yunikorn scheduler entered unhealthy state. 

Each suite can have an AfterEach, and the AfterEach code will run after each test in the suite.